app.rb

```diff
@@ -116,15 +139,14 @@ def create_rental
   end
 
   def list_rentals
-    if @people.empty?
+    if @rentals.empty?
       puts 'No data for people found.'
       return
     end
```

Comment:
Great job @Tamunokuro 🤝

- [ ] You have done well to implement the requirements for this milestone. However, your app doesn't seem to `Add a rental` successfully. I got the following error when I tried adding a rental.
![rental-error](https://user-images.githubusercontent.com/63917725/220761790-0dcd414b-9323-4ab0-93df-4f212721b695.PNG)